dfx-library/iirfilter.cpp

```diff
@@ -287,11 +300,11 @@ void dfx::Crossover::setFrequency(double inFrequency)
 {
 #if DFX_CROSSOVER_LINKWITZ_RILEY_MUSICDSP
 	// https://www.musicdsp.org/en/latest/Filters/266-4th-order-linkwitz-riley-filters.html
-	double const wc = 2. * dfx::math::kPi<double> * inFrequency;
+	double const wc = 2. * std::numbers::pi_v<double> * inFrequency;
 	auto const wc2 = wc * wc;
 	auto const wc3 = wc2 * wc;
 	auto const wc4 = wc2 * wc2;
-	double const k = wc / std::tan(dfx::math::kPi<double> * inFrequency / mSampleRate);
+	double const k = wc / std::tan(std::numbers::pi_v<double> * inFrequency / mSampleRate);
 	auto const k2 = k * k;
 	auto const k3 = k2 * k;
 	auto const k4 = k2 * k2;
```

Comment:
I dunno why it won't let me comment on the line, but the next line could be std::numbers::sqrt2_v, right? (Also if I understand correctly, "static" here could have synchronization overhead ("The initialization occurs exactly once" since C++11) because sqrt is not constexpr? I'm not sure it can evaluate it at compile time.)

Reply:
oh yeah, all true and great points, thanks for catching that!